pkg/transaction/eventgenerator.go

```diff
@@ -396,9 +396,10 @@ func updateWithProviderDetails(accessRequest *v1alpha1.AccessRequest, managedApp
 	}
 
 	summaryEvent.Product = &models.Product{
-		ID:      unknown,
-		Name:    unknown,
-		Version: unknown,
+		ID:          unknown,
+		Name:        unknown,
+		Version:     unknown,
```

Comment:
Can you name it VersionID to be explicit?

Reply:
yes :)